docs/src/prototypes/mentions/MentionsDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Dropdown, DropdownItemProps, Provider } from '@stardust-ui/react'
+
+import { MentionsContainerProps } from './MentionsEditor'
+
+const MentionsDropdown: React.FunctionComponent<MentionsContainerProps> = props => {
+  const { searchQuery, items, onOpenChange, onSearchQueryChange, onInputKeyDown } = props
+
+  const renderItem = React.useCallback(
+    (Item: React.ReactType, props: DropdownItemProps, boldFont: number) => {
+      const { header = '', ...rest } = props
+      const headerStr = String(header)
```

Comment:
The header is `ShorthandValue`. Please add check here if it is an object, we are interested in the content form the header's object.

Reply:
done, thanks